lib/bitstyles_phoenix/components/form.ex

```diff
@@ -37,38 +46,57 @@ defmodule BitstylesPhoenix.Form do
   @doc ~S"""
   Renders all types of `<input>` element, with the associated `<label>`s, and any errors for that field. Defaults to `type="text"`, and `maxlength="255"`.
   As with the various `text_input`, `number_input` helpers in `Phoenix.HTML.Form`, this expects the form & field as first parameters.
-  `opts` will be passed on to the respective underlying Phoenix form helper, with the following additional notes:
 
-  `form` — the form object this input is being rendered inside
-  `field` — the atom for the field you’re rendering
-  `opts[:type]` — the type of the input required, analogous to the input types in the HTML standard
-  `opts[:e2e_classname]` — A classname that will be applied to the input for testing purposes, only on integration env
-  `opts[:label]` — Override the default text to be used in the `<label>`
-  `opts[:hidden_label]` — The label element will be visually hidden, but still present in the DOM
+  ## Options
+  - `:type` — the type of the input (one of #{@input_mapping |> Map.keys() |> Enum.map(& &":#{&1}") |> Enum.join(", ")})
```

Comment:
Does this `#{@input_mapping |> Map.keys() |> Enum.map(& &":#{&1}") |> Enum.join(", ")}` gets interpreted into a nice text list?

Reply:
good catch :D it does not work yet apparently